beacon-chain/blockchain/receive_block.go

```diff
@@ -183,7 +184,7 @@ func (s *Service) ReceiveBlockNoPubsubForkchoice(ctx context.Context, block *eth
 func (s *Service) ReceiveBlockNoVerify(ctx context.Context, block *ethpb.SignedBeaconBlock) error {
 	ctx, span := trace.StartSpan(ctx, "beacon-chain.blockchain.ReceiveBlockNoVerify")
 	defer span.End()
-	blockCopy := proto.Clone(block).(*ethpb.SignedBeaconBlock)
+	blockCopy := proto.Clone(block).(*ethpb.SignedBeaconBlock) //stateTrie.CopySignedBeaconBlock(block)
```

Comment:
this is the confounding issue mentioned in comments.  Copy method commented out for now and will revisit on a subsequent PR

Reply:
Thanks again @nisdas for pointing out the proof copy issue.